GoogleForCreators/web-stories-wp
# Add back 'auto-draft' status

## Context
See https://github.com/GoogleForCreators/web-stories-wp/pull/12129#discussion_r948792595


## Summary
This PR removes rewriting `auto-draft` to `draft` when loading the story and updates the status checks where necessary.


## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions
1. Create a new story
2. Add some title, modify content, add some pages, etc. but **do not save**
3. Wait a minute or so for autosave to kick in
4. Go back to the "All Stories" page and ignore the warning about unsaved content
5. You should not see your story in the list (**<- this is new**)
6. Open the editor again
7. You should see a message about the version in your browser being newer than what you have, and that you can restore that version
8. Restore the backup and you should see the content again that you've added in step (2)





This PR can be tested by following these steps:

1.


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #
